packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -159,6 +159,7 @@ export interface ISemanticColors {
   menuItemBackgroundHovered: string;
 
   /**
+   * @deprecated
```

Comment:
Actually, depending on what Ben says, we may or may not deprecate this, since it's still being used in ComboBox.

Reply:
Okay, leave this deprecated.

Please do one of two things:
1) Upon closing this PR, don't close the linked issue, but update it to say that ComboBoxes still need to be updated to the correct styles, matching what you have here.
2) Or, update ComboBoxes to match the changes you've made already.

Thanks!